

from PyQt5 import QtCore, QtGui, QtWidgets
from Insert_window import Ui_COFFEECO
from Product_Window import Ui_Product_window

espresso = []
frappe = []
milkshake = []


class Ui_MainWindow(object):
    
    def openWindow(self):
        self.window = QtWidgets.QMainWindow()
        self.ui = Ui_COFFEECO()
        self.ui.setupUi(self.window)
        self.window.show()

    def prodWindow(self):
        self.window = QtWidgets.QMainWindow()
        self.ui = Ui_Product_window()
        self.ui.setupUi(self.window)
        self.window.show()

    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.setEnabled(True)
        MainWindow.resize(967, 720)
        MainWindow.setFixedSize(967, 720)
        self.centralwidget = QtWidgets.QWidget(MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.title = QtWidgets.QFrame(self.centralwidget)
        self.title.setGeometry(QtCore.QRect(0, 0, 971, 201))
        self.title.setStyleSheet("background-color: rgb(85, 0, 0);")
        self.title.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.title.setFrameShadow(QtWidgets.QFrame.Raised)
        self.title.setObjectName("title")
        self.line = QtWidgets.QFrame(self.title)
        self.line.setGeometry(QtCore.QRect(0, 179, 971, 31))
        self.line.setFrameShadow(QtWidgets.QFrame.Raised)
        self.line.setLineWidth(0)
        self.line.setMidLineWidth(26)
        self.line.setFrameShape(QtWidgets.QFrame.HLine)
        self.line.setObjectName("line")
        self.title_frame = QtWidgets.QFrame(self.title)
        self.title_frame.setGeometry(QtCore.QRect(190, 30, 561, 131))
        self.title_frame.setStyleSheet("border-color: rgb(0, 0, 0);\n"
"background-color: rgb(168, 75, 43);")
        self.title_frame.setFrameShape(QtWidgets.QFrame.Box)
        self.title_frame.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.title_frame.setLineWidth(4)
        self.title_frame.setObjectName("title_frame")
        self.business_logo = QtWidgets.QLabel(self.title_frame)
        self.business_logo.setGeometry(QtCore.QRect(20, 10, 111, 111))
        self.business_logo.setText("")
        self.business_logo.setPixmap(QtGui.QPixmap("business_logo.png"))
        self.business_logo.setScaledContents(True)
        self.business_logo.setObjectName("business_logo")
        self.business_title = QtWidgets.QLabel(self.title_frame)
        self.business_title.setGeometry(QtCore.QRect(140, 20, 411, 61))
        font = QtGui.QFont()
        font.setFamily("RomanC")
        font.setPointSize(48)
        font.setBold(True)
        font.setItalic(False)
        font.setUnderline(False)
        font.setWeight(75)
        font.setStrikeOut(False)
        self.business_title.setFont(font)
        self.business_title.setLayoutDirection(QtCore.Qt.LeftToRight)
        self.business_title.setFrameShape(QtWidgets.QFrame.NoFrame)
        self.business_title.setObjectName("business_title")
        self.business_subtitle = QtWidgets.QLabel(self.title_frame)
        self.business_subtitle.setGeometry(QtCore.QRect(140, 80, 411, 21))
        font = QtGui.QFont()
        font.setPointSize(10)
        self.business_subtitle.setFont(font)
        self.business_subtitle.setObjectName("business_subtitle")
        self.header = QtWidgets.QFrame(self.centralwidget)
        self.header.setGeometry(QtCore.QRect(0, 200, 971, 521))
        self.header.setStyleSheet("background-color: rgb(99, 43, 5);\n"
"")
        self.header.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.header.setFrameShadow(QtWidgets.QFrame.Raised)
        self.header.setObjectName("header")
        self.label_inventory = QtWidgets.QLabel(self.header)
        self.label_inventory.setGeometry(QtCore.QRect(40, 0, 291, 31))
        self.label_inventory.setStyleSheet("color: rgb(255, 255, 255);\n"
"font: 75 18pt \"Cambria\";")
        self.label_inventory.setObjectName("label_inventory")
        self.categories = QtWidgets.QFrame(self.header)
        self.categories.setGeometry(QtCore.QRect(30, 30, 901, 431))
        self.categories.setStyleSheet("background-color: rgb(99, 43, 5);")
        self.categories.setFrameShape(QtWidgets.QFrame.NoFrame)
        self.categories.setFrameShadow(QtWidgets.QFrame.Plain)
        self.categories.setObjectName("categories")
        self.horizontalLayout = QtWidgets.QHBoxLayout(self.categories)
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.espresso_frame = QtWidgets.QFrame(self.categories)
        self.espresso_frame.setStyleSheet("background-color: rgb(168, 75, 43);")
        self.espresso_frame.setFrameShape(QtWidgets.QFrame.Box)
        self.espresso_frame.setFrameShadow(QtWidgets.QFrame.Raised)
        self.espresso_frame.setLineWidth(3)
        self.espresso_frame.setObjectName("espresso_frame")
        self.espresso_title = QtWidgets.QLabel(self.espresso_frame)
        self.espresso_title.setGeometry(QtCore.QRect(90, 30, 181, 61))
        self.espresso_title.setAutoFillBackground(False)
        self.espresso_title.setStyleSheet("font: 75 36pt \"Script MT Bold\";")
        self.espresso_title.setObjectName("espresso_title")
        self.espresso_icon = QtWidgets.QLabel(self.espresso_frame)
        self.espresso_icon.setGeometry(QtCore.QRect(30, 30, 51, 61))
        self.espresso_icon.setText("")
        self.espresso_icon.setPixmap(QtGui.QPixmap("espresso_icon.png"))
        self.espresso_icon.setScaledContents(True)
        self.espresso_icon.setObjectName("espresso_icon")
        self.list_espresso = QtWidgets.QListWidget(self.espresso_frame)
        self.list_espresso.setGeometry(QtCore.QRect(20, 120, 256, 261))
        self.list_espresso.setStyleSheet("font: 16pt \"Cascadia Mono\";")
        font = QtGui.QFont()
        font.setPointSize(12)
        self.list_espresso.setFont(font)
        self.list_espresso.setFrameShape(QtWidgets.QFrame.Box)
        self.list_espresso.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.list_espresso.setObjectName("list_espresso")
        self.horizontalLayout.addWidget(self.espresso_frame)
        self.frappe_frame = QtWidgets.QFrame(self.categories)
        self.frappe_frame.setStyleSheet("background-color: rgb(168, 75, 43);\n"
"")
        self.frappe_frame.setFrameShape(QtWidgets.QFrame.Box)
        self.frappe_frame.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frappe_frame.setLineWidth(3)
        self.frappe_frame.setObjectName("frappe_frame")
        self.frappe_title = QtWidgets.QLabel(self.frappe_frame)
        self.frappe_title.setGeometry(QtCore.QRect(100, 30, 141, 61))
        self.frappe_title.setAutoFillBackground(False)
        self.frappe_title.setStyleSheet("font: 75 36pt \"Script MT Bold\";")
        self.frappe_title.setObjectName("frappe_title")
        self.frappe_icon = QtWidgets.QLabel(self.frappe_frame)
        self.frappe_icon.setGeometry(QtCore.QRect(36, 30, 61, 61))
        self.frappe_icon.setText("")
        self.frappe_icon.setPixmap(QtGui.QPixmap("frappe_icon.png"))
        self.frappe_icon.setScaledContents(True)
        self.frappe_icon.setObjectName("frappe_icon")
        self.list_frappe = QtWidgets.QListWidget(self.frappe_frame)
        self.list_frappe.setGeometry(QtCore.QRect(20, 120, 256, 261))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.list_frappe.setFont(font)
        self.list_frappe.setFrameShape(QtWidgets.QFrame.Box)
        self.list_frappe.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.list_frappe.setObjectName("list_frappe")
        self.list_frappe.setStyleSheet("font: 16pt \"Cascadia Mono\";")
        self.horizontalLayout.addWidget(self.frappe_frame)
        self.milkshake_frame = QtWidgets.QFrame(self.categories)
        self.milkshake_frame.setStyleSheet("background-color: rgb(168, 75, 43);\n"
"")
        self.milkshake_frame.setFrameShape(QtWidgets.QFrame.Box)
        self.milkshake_frame.setFrameShadow(QtWidgets.QFrame.Raised)
        self.milkshake_frame.setLineWidth(3)
        self.milkshake_frame.setObjectName("milkshake_frame")
        self.milkshake_title = QtWidgets.QLabel(self.milkshake_frame)
        self.milkshake_title.setGeometry(QtCore.QRect(60, 41, 211, 41))
        font = QtGui.QFont()
        font.setFamily("Script MT Bold")
        font.setPointSize(36)
        font.setBold(False)
        font.setItalic(False)
        font.setWeight(9)
        self.milkshake_title.setFont(font)
        self.milkshake_title.setAutoFillBackground(False)
        self.milkshake_title.setStyleSheet("font: 75 36pt \"Script MT Bold\";")
        self.milkshake_title.setObjectName("milkshake_title")
        self.milkshake_icon = QtWidgets.QLabel(self.milkshake_frame)
        self.milkshake_icon.setGeometry(QtCore.QRect(10, 30, 61, 61))
        self.milkshake_icon.setText("")
        self.milkshake_icon.setPixmap(QtGui.QPixmap("milkshake_icon.png"))
        self.milkshake_icon.setScaledContents(True)
        self.milkshake_icon.setObjectName("milkshake_icon")
        self.list_milkshake = QtWidgets.QListWidget(self.milkshake_frame)
        self.list_milkshake.setGeometry(QtCore.QRect(20, 120, 256, 261))
        font = QtGui.QFont()
        font.setPointSize(12)
        self.list_milkshake.setFont(font)
        self.list_milkshake.setFrameShape(QtWidgets.QFrame.Box)
        self.list_milkshake.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.list_milkshake.setObjectName("list_milkshake")
        self.list_milkshake.setStyleSheet("font: 16pt \"Cascadia Mono\";")
        self.milkshake_icon.raise_()
        self.milkshake_title.raise_()
        self.list_milkshake.raise_()
        self.horizontalLayout.addWidget(self.milkshake_frame)
        self.footer = QtWidgets.QFrame(self.header)
        self.footer.setGeometry(QtCore.QRect(-10, 450, 961, 61))
        self.footer.setStyleSheet("background-color: rgb(99, 43, 5);\n"
"")
        self.footer.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.footer.setFrameShadow(QtWidgets.QFrame.Raised)
        self.footer.setObjectName("footer")
        self.insert_button = QtWidgets.QPushButton(self.footer)
        self.insert_button.setGeometry(QtCore.QRect(310, 20, 171, 31))
        self.insert_button.setStyleSheet("background-color: rgb(255, 85, 0);\n"
"font: 11pt \"Rockwell\";")
        self.insert_button.setObjectName("insert_button")
        self.insert_button.clicked.connect(self.openWindow)
        self.insert_button.clicked.connect(MainWindow.close)
        self.view_button = QtWidgets.QPushButton(self.footer)
        self.view_button.setGeometry(QtCore.QRect(530, 20, 171, 31))
        self.view_button.setStyleSheet("background-color: rgb(255, 85, 0);\n"
"font: 11pt \"Rockwell\";")
        self.view_button.setObjectName("view_button")
        self.view_button.clicked.connect(self.prodWindow)
        self.view_button.clicked.connect(MainWindow.close)
        self.searchframe = QtWidgets.QFrame(self.header)
        self.searchframe.setGeometry(QtCore.QRect(720, 10, 241, 21))
        self.searchframe.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.searchframe.setFrameShadow(QtWidgets.QFrame.Raised)
        self.searchframe.setObjectName("searchframe")
        self.search_line = QtWidgets.QLineEdit(self.searchframe)
        self.search_line.setGeometry(QtCore.QRect(0, 0, 201, 20))
        self.search_line.setStyleSheet("background-color: rgb(255, 255, 255);")
        self.search_line.setObjectName("search_line")
        self.search_icon = QtWidgets.QLabel(self.searchframe)
        self.search_icon.setGeometry(QtCore.QRect(180, 0, 16, 16))
        self.search_icon.setStyleSheet("background-color: rgb(255, 255, 255);")
        self.search_icon.setText("")
        self.search_icon.setPixmap(QtGui.QPixmap("search bar.png"))
        self.search_icon.setScaledContents(True)
        self.search_icon.setObjectName("search_icon")
        self.header.raise_()
        self.title.raise_()
        MainWindow.setCentralWidget(self.centralwidget)

        for x in espresso :
            self.list_espresso.addItem(str(x))

        for x in frappe :
            self.list_frappe.addItem(str(x))

        for x in milkshake :
            self.list_milkshake.addItem(str(x))

        self.retranslateUi(MainWindow)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "MainWindow"))
        self.business_title.setText(_translate("MainWindow", "COFFEE.CO "))
        self.business_subtitle.setText(_translate("MainWindow", "A PROJECT BY: BURATO, FAJARDO, SEGUNDO, SIBAYAN, VILLANUEVA"))
        self.label_inventory.setText(_translate("MainWindow", "Inventory List Application"))
        self.espresso_title.setText(_translate("MainWindow", "Espresso"))
        self.list_espresso.setSortingEnabled(True)
        self.frappe_title.setText(_translate("MainWindow", "Frappe"))
        self.list_frappe.setSortingEnabled(True)
        self.milkshake_title.setText(_translate("MainWindow", "Milkshake"))
        self.list_milkshake.setSortingEnabled(True)
        self.insert_button.setText(_translate("MainWindow", "Insert"))
        self.view_button.setText(_translate("MainWindow", "View Products"))
        self.search_line.setPlaceholderText(_translate("MainWindow", "Search here"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    MainWindow = QtWidgets.QMainWindow()
    MainWindow.setWindowIcon(QtGui.QIcon('icon_logo.png'))
    ui = Ui_MainWindow()
    ui.setupUi(MainWindow)
    MainWindow.show()
    sys.exit(app.exec_())
